packages/plugin-core/src/commands/GotoNote.ts

```diff
@@ -274,13 +275,33 @@ export class GotoNoteCommand extends BasicCommand<
     let pos: undefined | Position;
     const out = await this.extension.pauseWatchers<GoToNoteCommandOutput>(
       async () => {
+        const existing = await client.getNoteByPath({
```

Comment:
confused about this line. doesn't it get the path of the note that is being created, not the existing note?

Reply:
The variable is named misleadingly - it's basically querying the engine if a note named ${qs} - 'createIfNew' is set to false.  It's used below, if `if (!existing.data?.note)`, i.e, if there isn't an existing note, then send the telem marker.